web/src/main/java/org/springframework/security/web/jackson2/WebServletJackson2Module.java

```diff
@@ -58,7 +57,6 @@ public void setupModule(SetupContext context) {
 		context.setMixInAnnotations(SavedCookie.class, SavedCookieMixin.class);
 		context.setMixInAnnotations(DefaultSavedRequest.class, DefaultSavedRequestMixin.class);
 		context.setMixInAnnotations(WebAuthenticationDetails.class, WebAuthenticationDetailsMixin.class);
-		context.setMixInAnnotations(SwitchUserGrantedAuthority.class, SwitchUserGrantedAuthorityMixIn.class);
```

Comment:
While I see your point, we can't do this since it's a public class and applications may be using this separately from `WebJackson2Module`. As such, will you please leave it in place here?